client/state/action-types.ts

```diff
@@ -373,6 +373,12 @@ export const JETPACK_CONNECTION_STATUS_REQUEST_FAILURE =
 	'JETPACK_CONNECTION_STATUS_REQUEST_FAILURE';
 export const JETPACK_CONNECTION_STATUS_REQUEST_SUCCESS =
 	'JETPACK_CONNECTION_STATUS_REQUEST_SUCCESS';
+export const JETPACK_CONNECTION_HEALTH_REQUEST = 'JETPACK_CONNECTION_HEALTH_REQUEST';
+export const JETPACK_CONNECTION_HEALTH_REQUEST_RECEIVE = 'JETPACK_CONNECTION_HEALTH_RECEIVE';
+export const JETPACK_CONNECTION_HEALTH_REQUEST_SUCCESS =
```

Comment:
Since this is only used for loading progress, and I suggested removing that, we should be able to ditch the `JETPACK_CONNECTION_HEALTH_REQUEST_SUCCESS` action type completely.

Reply:
I use that in the jetpack health banner as I mentioned below. Any ideas on how / if we can do better?

Reply:
I left a comment above on how to get rid of its sole, and IMHO unnecessary usage.

Reply:
I see that now! I remove the unnecessary code! Thanks!